File: feedsphere/check-schema.js
const { db } = require('./lib/db');
require('dotenv').config({ path: '.env.local' });

async function checkSchema() {
  try {
    const res = await db.query(`
      SELECT column_name, data_type 
      FROM information_schema.columns 
      WHERE table_name = 'posts'
    `);
    console.log('Columns in posts table:');
    res.rows.forEach(row => console.log(`- ${row.column_name}: ${row.data_type}`));
  } catch (e) {
    console.error('Error checking schema:', e);
  } finally {
    process.exit();
  }
}

checkSchema();

File: tmp/check_schema.py
from pipeline.db import db

def check_schema():
    query = "SELECT column_name FROM information_schema.columns WHERE table_name = 'debates'"
    columns = db.fetch_all(query)
    print("Debates table columns:")
    for col in columns:
        print(f"- {col['column_name']}")

if __name__ == "__main__":
    check_schema()
